src/occt_wrapper/CMakeLists.txt

```diff
@@ -19,14 +19,9 @@ include(GenerateExportHeader)
 
 generate_export_header(OCCTWrapper)
 
-find_package(OpenCASCADE 7.6.1 REQUIRED)
+find_package(OpenCASCADE REQUIRED)
 
 set(OCCT_LIBS
-    TKXDESTEP
```

Comment:
There is only a single lib needed as it pulls in all the required ones, even with 7.6, see:

https://github.com/prusa3d/PrusaSlicer/pull/13761

Reply:
See also https://github.com/prusa3d/PrusaSlicer/issues/14395

Reply:
This reliance on transitive dependencies works for dynamic linking but not for static linking as Prusa usually does. So we either need to leave the list as it currently is or add some if statements on the version number to select all relevant libraries for each version when libraries change.